cli.js

```diff
@@ -41,5 +45,7 @@ if (process.stdin.isTTY) {
 
 	init(cli.input);
 } else {
-	process.stdin.pipe(isProgressive.stream(init));
+	isProgressive.stream(process.stdin).then(p => {
+		log(p, 'file');
+	});
```

Comment:
I believe the original code was not working because the callback if `isProgressive.stream` returned `true` or `false` and not an `input` file. So had to do some refactoring here.

The thing is, I can't seem to find a way to retrieve the filename of `process.stdin`. Is that even possible? So if a file is streamed in, the only thing it shows right now is `✔ file`.